transformations/summarization_transformation/transformation.py

```diff
@@ -0,0 +1,188 @@
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import spacy
+
+"""
+The code is partially adapted and modified from https://github.com/NSchrading/intro-spacy-nlp
+"""
+
+
+class Summarization(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+    ]
+    languages = ["en"]
+
+    def __init__(self):
+        # super().__init__(seed, max_outputs=max_outputs)
+        self.nlp = spacy.load('en_core_web_sm', disable=['ner','textcat'])
+        self.dicts ={
+            'SUBJECTS': ["nsubj", "nsubjpass", "csubj", "csubjpass", "agent", "expl"],
+            'OBJECTS': ["dobj", "dative", "attr", "oprd", "acomp", "advcl", "advmod", "amod", "appos", "nn", "nmod", "ccomp", "complm", "hmod", "infmod", "xcomp", "rcmod", "poss"," possessive"],
```

Comment:
Some comments or references/ pointers on these various categories' abbreviations will be helpful.

Reply:
Please kindly find the reference for the tags we used in the code [here](https://downloads.cs.stanford.edu/nlp/software/dependencies_manual.pdf)

Reply:
Linking to the document in the readme would be a good idea.